controllers/api.js

```diff
@@ -216,6 +216,88 @@ exports.getGithub = async (req, res, next) => {
   }
 };
 
+/**
+ * GET /api/imgur
+ * Imgur API example.
+ */
+exports.getImgur = async (req, res, next) => {
+  try {
+    const clientId = process.env.IMGUR_CLIENT_ID;
+
+    if (!clientId) {
+      return res.render('api/imgur', {
+        title: 'Imgur API',
+        error: 'IMGUR_CLIENT_ID environment variable is not set. Please obtain a Client ID from https://api.imgur.com/oauth2/addclient',
+        images: [],
+        galleries: [],
+        tags: [],
+      });
+    }
+
+    const headers = {
+      Authorization: `Client-ID ${clientId}`,
+      'Content-Type': 'application/json',
+    };
+
+    try {
+      // Fetch gallery images (top viral images)
+      const [galleryResponse, tagsResponse] = await Promise.all([
+        fetch('https://api.imgur.com/3/gallery/hot/viral/0.json?showViral=true&mature=false&album_previews=true', {
```

Comment:
Let's use an image set that has a lower chance of having inappropriate for a professional setting like:
https://imgur.com/t/programming
https://imgur.com/t/technology
https://imgur.com/t/engineering
https://imgur.com/t/sciencefair
...